app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,22 @@ def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
   end
+
+  def new
+    @user = User.find(params[:user_id])
+    @post = Post.new
+  end
+
+  def create
+    @user = current_user
+    @post = Post.new(params.require(:post).permit(:title, :text))
```

Comment:
**OPTIONAL**

- [ ] It is always good practice to put `params.require(:post).permit(:title, :text)` in a method and make the method private. This is a security feature in rails. To learn more about this kindly refer to this[ lesson](`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides`)